### PR TITLE
Create spike.md

### PR DESCRIPTION
**Pull request by amoss**
##### Summary

 Some structure for spike tasks.

### CODE REVIEW

**Comment by cakrit** [comment]
I think it could use a few more words (perhaps explanation comments on what needs to be written and when things need to be checked), it looks a bit cryptic to me...

**Comment by amoss** [comment]
> I think it could use a few more words (perhaps explanation comments on what needs to be written and when things need to be checked), it looks a bit cryptic to me...

I have made it very wordy.

File: .github/ISSUE_TEMPLATE/spike.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Questions To Investigate
+
+Please explain the area that will be investigated during the spike with concrete
+questions that will answered.
```
Comment (cakrit) [inline .github/ISSUE_TEMPLATE/spike.md:4]: that will **be** answered

Comment (amoss) [reply .github/ISSUE_TEMPLATE/spike.md:4]: Unbelievable! I was literally one word short of wordiness :).

Comment (shortpatti) [reply .github/ISSUE_TEMPLATE/spike.md]: Explain the area to be investigated during the spike with concrete questions you want answered. 

(Just a rule of thumb in technical docs, don't use "please".)

**Comment by cosmix** [comment]
@amoss perhaps you might want to revisit this one and get it in.

**Comment by amoss** [comment]
This is irrelevant with the change in process.

**Pull request closed without merging**